src/components/members.component.js

```diff
@@ -29,45 +23,94 @@ type Props = {
 
 const avatarSize = 30;
 
-const styles = StyleSheet.create({
-  container: {
-    marginTop: 30,
-  },
-  avatarContainer: {
-    backgroundColor: colors.greyLight,
-    borderRadius: avatarSize / 2,
-    width: avatarSize,
-    height: avatarSize,
-  },
-  avatar: {
-    borderRadius: avatarSize / 2,
-    height: avatarSize,
-    width: avatarSize,
-  },
-  list: {
-    marginTop: 0,
-  },
-  sectionTitle: {
-    color: colors.black,
-    ...fonts.fontPrimaryBold,
-    marginBottom: 10,
-    paddingLeft: 15,
-  },
-  sectionTitleSmall: {
-    color: colors.primaryDark,
-    ...fonts.fontPrimarySemiBold,
-    marginBottom: 10,
-    paddingLeft: 15,
-  },
-  flatList: {
-    paddingHorizontal: 15,
-  },
-  scrollGradient: {
-    position: 'absolute',
-    width: 15,
-    height: avatarSize,
-  },
-});
+const Container = styled.View`
+  margin-top: 30;
+  ${props => props.containerStyle};
+`;
+
+const AvatarContainer = styled.TouchableHighlight`
+  background-color: ${colors.greyLight};
+  border-radius: ${avatarSize / 2};
+  width: ${avatarSize};
+  height: ${avatarSize};
+  ${props => props.style};
+`;
+
+const Avatar = styled.Image`
+  border-radius: ${avatarSize / 2};
+  height: ${avatarSize};
+  width: ${avatarSize};
+`;
+
+type StyledListProps = {
+  style: Object,
+  children: Element,
+};
+
+const StyledList = ({ style, children }: StyledListProps) => (
+  <List containerStyle={style}>{children}</List>
+);
+
+const NoMembersList = styled(StyledList)`
+  margin-top: 0;
+`;
+
```

Comment:
Instead of creating a new component, once #619 is merged you can do this as follow:

```js
const NoMembersList = styled(List).attrs({
  containerStyle: {
    margin-top: 0;
  }
})``;
```